validator/db/kv/prune_attester_protection_test.go

```diff
@@ -39,7 +39,7 @@ func TestPruneAttestations_NoPruning(t *testing.T) {
 }
 
 func TestPruneAttestations_OK(t *testing.T) {
-	numKeys := uint64(2048)
+	numKeys := uint64(64)
```

Comment:
Please explain this change

Reply:
Same functionality and thing we are testing. However, 2048 takes a long time for test set up. If we want to benchmark things, we have a benchmark function we can use. The unit test is just to check things work as expected